streams/src/test/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImplTest.java

```diff
@@ -777,12 +777,12 @@ public void testLatencyMetrics() {
         final String entity = "entity";
         final String operation = "put";
 
-        final Sensor sensor1 = streamsMetrics.addLatencyAndThroughputSensor(scope, entity, operation, RecordingLevel.DEBUG);
```

Comment:
You can also remove the `@SuppressWarnings("deprecation")` at the top of the method.